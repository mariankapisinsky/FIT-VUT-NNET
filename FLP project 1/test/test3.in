E,T,F
p,x,l,r
E
E->EpT
E->T
T->TxF
T->F
F->lEr
F->i
