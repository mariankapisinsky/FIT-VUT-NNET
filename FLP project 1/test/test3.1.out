E,T,F
p,x,l,r
E
E->EpT
E->TxF
E->lEr
E->i
T->TxF
T->lEr
T->i
F->lEr
F->i
